### PR TITLE
feat: push 'use operator or not' to collector

### PR DESCRIPTION
**Pull request by CeerDecy**
#### What this PR does / why we need it:
push 'use operator or not' to collector

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/all?id=613202&iterationID=12783&type=TASK)

#### Specified Reviewers:

/assign @iutx 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |     push 'use operator or not' to collector         |
| 🇨🇳 中文    |       输出"Addon部署是否使用operator"到collector      |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: cmd/erda-server/bootstrap.yaml [hunk 1]
```diff
@@ -377,7 +377,7 @@ cmp:
   # steve cache total size (MB), default 64MB
   cache_size: ${CMP_STEVE_CACHE_SIZE:64}
 grpc-client@erda.core.monitor.metric:
-  addr: "${MONITOR_GRPC_ADDR:monitor:7080}"xwxxx
```
Comment (iutx) [inline cmd/erda-server/bootstrap.yaml:380]: amazing

File: internal/tools/orchestrator/services/addon/addon_handler.go [hunk 2]
```diff
@@ -1149,6 +1149,9 @@ func (a *Addon) basicAddonDeploy(addonIns *dbclient.AddonInstance, addonInsRouti
 	}
 	logrus.Infof("sending addon creating request, request body: %+v", *addonCreateReq)
 
+	_, ok := addonCreateReq.DiceYml.Meta["USE_OPERATOR"]
+	a.pushLog(fmt.Sprintf("use operator: %v", ok), params)
```
Comment (iutx) [inline internal/tools/orchestrator/services/addon/addon_handler.go:1153]: I think we can add more target addon info, such as version, name,  before submitting the `ServiceGroup`.

**Review by iutx — COMMENTED** [review]
@CeerDecy

**Review by sfwn — APPROVED** [review]

**Pull request merged**